Show first and last pages on relationships (#23063)
directus/directus: .changeset/young-phones-shop.md

---
"@directus/app": patch
---

> Show first and last pages on pagination of relationship interfaces

Reviewer: ```suggestion
Added first and last page buttons to pagination in relational interfaces
```
